spec/swagger.yaml

```diff
@@ -2739,9 +2739,9 @@ paths:
                   issueSeverityLevelCode: LOW
   '/issues-api/v2/projects/{projectId}/issues/{issueUid}/answered':
     put:
-      summary: Change issue attribute "answered"
+      summary: Update the "answered" attribute of an issue
```

Comment:
the issue?

Reply:
@hchaps should we quote attributes like this? Just looking for a general practice / guideline?

Reply:
@jones-smartling : I'd maybe go with something like: 
- Update issue attribute (answered)

And for the other one:
- Update issue attribute (severity level)